vector/src/main/res/xml/vector_settings_preferences.xml

```diff
@@ -22,6 +22,11 @@
             android:title="@string/settings_theme"
             app:iconSpaceReserved="false" />
 
+        <im.vector.app.core.preference.VectorSwitchPreference
+            android:defaultValue="false"
+            android:key="SETTINGS_INTERFACE_BUBBLE_KEY"
+            android:title="@string/message_bubbles" />
```

Comment:
Better to move this in the `timeline` section

Reply:
done